minidump/src/minidump.rs

```diff
@@ -5182,6 +5182,8 @@ impl MinidumpCrashpadInfo {
     }
 }
 
+pub type MmapMinidump<'a> = Minidump<'a, Mmap>;
```

Comment:
Can you put a doc comment on here, or on `Minidump::read_path` (or both) that tells users how to use this?

Reply:
Interestingly, the doctest uses Minidump::read_path directly, and relies on type inference, not requiring to use Mmap at all. Only minidump-writer uses the type because it stores it in a struct.